### PR TITLE
Episode 02 improve exercises

### PR DESCRIPTION
**Pull request by JulianKarlBauer**
- Link exercises
- Use GitHub format of rendered views
- Add call out on cheatsheet

### CODE REVIEW

File: _episodes/02-markdown.md [hunk 1]
```diff
@@ -119,123 +119,82 @@ Let's do some exercises to learn more about structuring and emphasis.
...
-
+You can use single backticks to call out code, e.g. `print('Hello world')` and
+use triple backticks to format larger code snippets, creating code blocks.
 
 > ## Add Code
-> Fill in the blanks below to create the rendered view below this line:
+> Fill in the blanks ( __ ) below to create the following view
 >
-> ---
+> ![Preview tab with blanks before code and code blocks](../fig/exercise_add_code.png)
```
Comment (tobyhodges) [inline _episodes/02-markdown.md:187]: I'm confused about why the preview includes the blanks? Shouldn't this exercise include a block of Markdown with blanks, and a preview of how the completed Markdown should render, after the blanks have been filled in?

File: _episodes/02-markdown.md [hunk 2]
```diff
@@ -251,34 +210,38 @@ Let's do some exercises to learn more about structuring and emphasis.
...
-> > <a href=https://carpentries.org/>This is a link</a>
-> > </body>
+> <a href=https://carpentries.org/>This is a link</a>
+> </body>
 > </html>
 > ```
 > ~~~
 > {: .source }
 >
+> guess in which programming language each snippet is written
```
Comment (tobyhodges) [inline _episodes/02-markdown.md]:
```suggestion
> then guess in which programming language each snippet is written,
```

File: _episodes/02-markdown.md [hunk 3]
```diff
@@ -296,34 +259,31 @@ You can define either
...
 > 1. [___](https://carpentries.org/)
 > 2. [___]___case-InSeNsiTiVe-reference-tag___
 >
 > [case-insensitive-reference-tag]: https://carpentries.org/
 > ~~~
 > {: .source }
 >
+> to create the following rendered view:
+>
+> ![Preview tab of code blocks with syntax highlighting](../fig/exercise_links.png)
```
Comment (tobyhodges) [inline _episodes/02-markdown.md]: Please update the alt text here. At the moment it describes the image (of rendered links) as showing code blocks with syntax highlighting.

File: _episodes/02-markdown.md [hunk 4]
```diff
@@ -296,34 +259,31 @@ You can define either
...
-> Examples:
+> Example links:
 > 1. [___](https://carpentries.org/)
 > 2. [___]___case-InSeNsiTiVe-reference-tag___
 >
 > [case-insensitive-reference-tag]: https://carpentries.org/
 > ~~~
 > {: .source }
 >
+> to create the following rendered view:
```
Comment (tobyhodges) [inline _episodes/02-markdown.md]:
```suggestion
> to create the following rendered view, where both links point to <https://carpentries.org/>:
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thanks, @JulianKarlBauer! This looks good but needs a small number of changes before merging.

File: _episodes/02-markdown.md (hunk 4 above)
Comment (tobyhodges) [reply _episodes/02-markdown.md]: Adding this to remove ambiguity, as the links aren't clickable in the image.

File: _episodes/02-markdown.md (hunk 3 above)
Comment (JulianKarlBauer) [reply _episodes/02-markdown.md]: Thank you, see 63ecec4 for a fix of this copy paste mistake.

File: _episodes/02-markdown.md (hunk 1 above)
Comment (JulianKarlBauer) [reply _episodes/02-markdown.md:187]: Thanks, I thought this exercise could do without code-template, but your confusion proofed that I was wrong.
I adopted the format used in the other "fill the blanks" exercises in 
7bb3b53  and agree, that this is much better.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**